demo/README.md

```diff
@@ -38,7 +38,8 @@ docker run --rm -it \
     -v /tmp/.X11-unix:/tmp/.X11-unix \
     --device=/dev/video0:/dev/video0 \
     --ipc=host maskrcnn-benchmark \
-    python demo/webcam.py --min-image-size 300
+    python demo/webcam.py --min-image-size 300 \
```

Comment:
is this change necessary?

Reply:
yes the default is `../configs/caffe2/e2e_mask_rcnn_R_50_FPN_1x_caffe2.yaml` , while when running from docker and from `maskrcnn-benchmark` folder  it should be `configs/caffe2/e2e_mask_rcnn_R_50_FPN_1x_caffe2.yaml`

Reply:
alright, thanks!

Reply:
Note that the demo won't work if you run it from `maskrcnn-benchmark` folder due to the local includes, so this breaks in that case

Reply:
for example, trying to `import predictor` won't work if done from `maskrcnn-benchmark`, and it is explicitly mentioned in the readme that you should cd to `demo` first, so I'd vote for removing this change

Reply:
running the current command: 
```
docker run --rm -it \
    -e DISPLAY=${DISPLAY} \
    --privileged \
    -v /tmp/.X11-unix:/tmp/.X11-unix \
    --device=/dev/video0:/dev/video0 \
    --ipc=host maskrcnn-benchmark \
    python demo/webcam.py --min-image-size 300
```
throws the error `FileNotFoundError: [Errno 2] No such file or directory: '../configs/caffe2/e2e_mask_rcnn_R_50_FPN_1x_caffe2.yaml'`

this change is just to specify path to config file in this particular case: 
```
docker run --rm -it \
    -e DISPLAY=${DISPLAY} \
    --privileged \
    -v /tmp/.X11-unix:/tmp/.X11-unix \
    --device=/dev/video0:/dev/video0 \
    --ipc=host maskrcnn-benchmark \
    python demo/webcam.py --min-image-size 300 \
    --config-file configs/caffe2/e2e_mask_rcnn_R_50_FPN_1x_caffe2.yaml
```

it should not break any imports in the demo folder

Reply:
I just noticed that this is for the `docker` example.

I have never tried using it with docker, but I know for sure that without docker running the webcam demo code from `maskrcnn-benchmark` folder will not work, you need to be in the `demo` folder.

But maybe some paths are setup differently in the dockerfile.

Reply:
wierd, works fine for me outside of `demo` folder (not in docker) using python 3.6 :

```
python demo/webcam.py --min-image-size 300 --config-file configs/caffe2/e2e_mask_rcnn_R_50_FPN_1x_caffe2.yaml MODEL.DEVICE cpu
```